t/ui/13-admin.t

```diff
@@ -448,6 +448,29 @@ subtest 'job property editor' => sub() {
         is($driver->find_element_by_id('editor-carry-over-bugrefs')->is_selected(), 0, 'bug carry over disabled');
         is($driver->find_element_by_id('editor-description')->get_value(), 'Test group', 'description added');
     };
+
+    subtest 'edit the yaml' => sub() {
+        $driver->refresh();
+        my $form = $driver->find_element_by_id('editor-form');
+        ok($form->is_hidden(), 'editor form is hidden');
+        $driver->find_element_by_id('toggle-yaml-editor')->click();
+        ok($form->is_displayed(), 'editor form is shown');
+        ok($driver->find_element_by_id('update-template')->is_displayed(), 'update button is shown');
+        ok($form->child('.progress-indication')->is_hidden(), 'spinner is hidden');
+        my $yaml = $driver->execute_script('return editor.doc.getValue();');
+        ok($yaml =~ m/Cool Group/, 'YAML was fetched');
+        $driver->find_element_by_id('update-template')->click();
+        my $result = $form->child('.result');
+        is($result->get_text(), 'Applying changes...', 'update in progress');
```

Comment:
Like I explained earlier this won't work reliably. The 'Applying changes...' might have already been disappeared by the time this code is executed. Always keep in mind that there is no magic synchronization between the Perl code and chromedriver in general.

This would also explain why the test fails in Travis:

```
        #   Failed test 'update in progress'
        #   at ./t/ui/13-admin.t line 464.
        #          got: 'Preview of the YAML:
        # ---
        # Cool Group has been edited!:
        #   architectures: {}
        #   products: {}'
        #     expected: 'Applying changes...'
        # Looks like you failed 1 test of 8.
```